firmware/platformio.ini

```diff
@@ -34,7 +35,7 @@ lib_deps =
     ESP Async WebServer@1.2.3
     LoRa@0.7.2
     https://github.com/jgromes/RadioLib#3682c6c9215891e3afb7672f1235fde1c3bd75fd
-    https://github.com/sudomesh/LoRaLayer2#efaa3fa73e3c8f6a7c66e335873c0bd81cc865e3
+    https://github.com/code8buster/LoRaLayer2.git#esp32v3
```

Comment:
After merging both PRs, we should point this back at ""official" LoRaLayer2 latest commit.